README.md

```diff
@@ -133,6 +133,18 @@ EOF
 crontab crontab
 ```
 
+## running it locally using Docker
```

Comment:
very minor nits: mind capitalizing _running_ here? and _run it_ and _open it_ below